piqueserver/server.py

```diff
@@ -461,10 +471,11 @@ def _next_time_announce(self):
             if remaining < 10.001:
                 self.broadcast_chat('%s...' % int(round(remaining)))
             else:
-                self.broadcast_chat('%s seconds remaining.' % int(round(remaining)))
+                self.broadcast_chat('%s seconds remaining.' %
+                                    int(round(remaining)))
         else:
             self.broadcast_chat('%s minutes remaining.' %
-                           int(round(remaining / 60)))
+                                int(round(remaining / 60)))
```

Comment:
Some of these formatting changes are out-of-scope for this PR, assuming they were done automatically by your IDE

Reply:
These changes are actually made by autopep8 that tries to keep the formatting standards in pep8. Thats why i made it in a separete commit, but we can revert it anyway, i just kept it because it will need to get updated at some point